### ISSUE TITLE
WebUI - full address

### ISSUE THREAD

**Issue by benjiworld** [billy4479/telegram-storage#7]
Hello, I've been able (with some fixes) to execute and run via docker-compose the server. Sending /link to the group the temporary token is correctly received. But pointing to http://somehost:4479 gives this result:
{
  "error": "code=404, message=Not Found",
  "message": "Not Found"
}

What am I doing wrong?
Is there any deep page to reach, for example 

http://somehost:4479/somepageWebUI

Thank you advance, I'd really like to give your project a try...

**Comment by billy4479** [comment]
Hi,
it's been a loooong time since I've wrote any code for this project, I don't support it anymore and I should probably archive it.

Anyways, if I remember correctly, if you're using the default docker compose file you should then have a container for the backend and one for the frontend (and various others, it was a mess, I admit it).

- The backend is available on port 4479, on the "/api" route. If you're a little familiar with Go code you can check [api.go](https://github.com/billy4479/telegram-storage/blob/master/backend/api/api.go) to see the available endpoints.
- The frontend should be available on port 3000, and "/" should give you the landing page if I remember correctly.

Give it a try, feel free to do whatever you want with it. Let me know if you need any more help :)

**Comment by benjiworld** [comment]
Thank you for your quick answer...
I'm sorry you decided not to develop and mantain this project anymore, anyway, I've been able to reach the landing page at :3000 port, but from there the registration is not working.
I /link the TG group and I get the token, but from there nothing else...
Let me attach the servers status screenshots.
![1](https://github.com/billy4479/telegram-storage/assets/76999447/eacd397e-0fb4-4c7b-a3c0-65b5da78bcc2)
![2](https://github.com/billy4479/telegram-storage/assets/76999447/b082bb26-5c83-45a9-9e36-6101048c665a)
Thanks a lot again!

**Comment by billy4479** [comment]
The only thing i notice is that in the token there are some `\`. Try manually removing those.

The temporary token it's just a [JWT](https://jwt.io/), you should also be able to inspect it if you'd like, but it should never contain backslashes.

The ones you see in the server logs are there because of the formatting that's being used in telegram messages. If you copy the token as the bot sent it to you you should be able to register (the bot did reply to your "/link" command right? it should be in your private chat with the bot, not the group chat).

This is quite an old project of mine and there's definitely a lot to improve. I am graduating high school right now and I'm in the middle of the exams so I don't have much time to code at all, maybe next year in college I'll try to take the project forward, who knows :P

**Comment by benjiworld** [comment]
Yes, already noticed the slashes around, but I've used the token from the bot.. That was not the problem unfortunately.. So, focus your attention on the exams, and don't pay attention to other stuffs, I'll try to dig around in the while.. thank you for your help, wish you the best...
